aten/src/ATen/Declarations.cwrap

```diff
@@ -1515,7 +1515,6 @@
     - arg: THTensor* result
       output: True
       resize: [ [self,0], [self,1], [mat2,2] ]
-      cpu_zero: True
```

Comment:
What is `cpu_zero` supposed to do?

Reply:
For the out-of-place variant, it zeros the output that is passed in to the TH function if on CPU.

(Yes, we have a flag for that).